### PR TITLE
Fix long contract name issue

### PR DESCRIPTION
**Pull request by YegorSan**
This PR is related to #2527 issue.

## Motivation

View for long contract names was broken.

### Bug Fixes
Fixed long contract name issue. 
Screenshot (3)

Working now on Transaction Details page as bug appearing not only in internal transaction tile: 
Screenshot (4)

## Checklist for your PR

  - [ ] I added an entry to `CHANGELOG.md` with this PR
  - [ ] If I added new functionality, I added tests covering it.
  - [ ] If I fixed a bug, I added a regression test to prevent the bug from silently reappearing again.
  - [ ] I checked whether I should update the docs and did so if necessary
  - [ ] If I added/changed/removed ENV var, I should update the list of env vars in https://github.com/poanetwork/blockscout/blob/master/docs/env-variables.md to reflect changes in the table here https://poanetwork.github.io/blockscout/#/env-variables?id=blockscout-env-variables. I've set `master` in the `Version` column.
  - [ ] If I add new indices into DB, I checked, that they don't redundant with PGHero or other tools

### CODE REVIEW

File: apps/block_scout_web/assets/css/_layout.scss [hunk 1]
```diff
@@ -11,4 +11,11 @@
...
-}
\ No newline at end of file
+}
+  
+.truncated {
+    width: 200px;
+    white-space: nowrap;
+    overflow: hidden;
+    text-overflow: ellipsis;
+{
```
Comment (vbaranov) [inline apps/block_scout_web/assets/css/_layout.scss]: @YegorSan I believe, this is a typo. Please check tests after you make an update.

Comment (YegorSan) [reply apps/block_scout_web/assets/css/_layout.scss]: @vbaranov Ok Victor, I'll re check tests and push an update.

**Comment by YegorSan** [comment]
@vbaranov Hi Victor. Sorry for such a mistake. I've fixed braces typo and build test is passing now. Will push a small fix for long contract name in transaction tile in mobile view in 5 minutes.

**Comment by vbaranov** [comment]
@YegorSan please attach screenshots of how the screens will look with this update

**Comment by YegorSan** [comment]
@vbaranov Hello Victor. Unfortunately, I can not reproduce this issue locally (can't find contract with such a long name), so I can not provide the screenshot with the performed changes. I've worked on that issue in chrome inspector on stg app, and then performed all needed changes locally.

**Comment by igorbarinov** [comment]
> @YegorSan
> 
> Screenshot 2019-10-08 at 22 42 02

how it's implemented in etherscan @vbaranov

**Comment by YegorSan** [comment]
@vbaranov Hi Victor. I've fixed issue with overflow: ellipsis, as it not worked as it expected with flexbox (not showing three dots if truncated). Now I've added display: flex to the parent span element and looks like it works fine now. I want to add that I've deleted unnecessary truncation for contract name with the standard length:

Screenshot (7)

For testing purposes I've hardcoded contract name value, and it will truncate string if the length is larger than standard legth:

Screenshot (8)
 
Could you please review? Thank's in advance.

**Comment by vbaranov** [comment]
>how it's implemented in etherscan @vbaranov

@igorbarinov Etherscan doesn't show Contract name on the tx tile at all

**Comment by YegorSan** [comment]
@vbaranov Checking now.

**Comment by igorbarinov** [comment]
> > how it's implemented in etherscan @vbaranov
> 
> @igorbarinov Etherscan doesn't show Contract name on the tx tile at all

IMHO it should be with `..` like with addresses 

`Longcontract` should be `Longco..ct`

**Pull request closed without merging**